include/klee/BoundedMergeHandler.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
Your comments added to this PR are excellent.  Please add them here, in doxygen format. 
Also, please add the standard header with the KLEE license to this file.